src/benchmarks/gc/GC.Infrastructure/Versions.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project> 
+  <PropertyGroup>
+    <!-- To use a custom TraceEvent library, specify CustomTraceEvent to true and the PerfViewPath below -->
+    <CustomTraceEvent>false</CustomTraceEvent>
+    <PerfViewPath>$(MSBuildProjectDirectory)\..\..\..\..\..\..\PerfView</PerfViewPath>
+    <DefineConstants Condition="'$(CustomTraceEvent)' == 'true'">CUSTOM_TRACE_EVENT;$(DefineConstants)</DefineConstants>
```

Comment:
Is this necessary or can be removed? We hadn't needed it before.

Reply:
The whole point of this change is to make it possible to use a local version where we never able to before.